src/tx-builder/services/MerkleDistributor.ts

```diff
@@ -76,7 +78,7 @@ export default class MerkleDistributor extends BaseService<MerkleDistributorV1>
     return this.getContractInstance(this.merkleDistributorAddress);
   }
 
-  private async getActiveRootData(): Promise<ActiveRootDataAndHistory> {
+  public async getActiveRootData(): Promise<ActiveRootDataAndHistory> {
```

Comment:
Why did you make this `public`?

Reply:
I thought we should expose each part of `getActiveUsersInEpoch` but maybe that's dumb since the client is already public so anyone can see how the code is written